src/group/components/GroupEdit.vue

```diff
@@ -32,63 +38,83 @@
         <MarkdownInput
           v-if="!edit.isPlayground"
           v-model="edit.publicDescription"
-          icon="fas fa-fw fa-question"
-          :label="$t('GROUPINFO.TITLE')"
           :error="hasError('publicDescription')"
           :error-message="firstError('publicDescription')"
+          :label="$t('GROUPINFO.TITLE')"
+          icon="fas fa-fw fa-question"
+          outlined
+          :tmp-new-border-style="true"
+          class="q-mb-lg"
           @keyup.ctrl.enter="maybeSave"
         />
 
         <MarkdownInput
           v-model="edit.description"
-          icon="fas fa-fw fa-address-card"
-          :label="$t('GROUP.DESCRIPTION_VERBOSE')"
           :error="hasError('description')"
           :error-message="firstError('description')"
-          @keyup.ctrl.enter="maybeSave"
-        />
-
-        <MarkdownInput
-          v-if="!edit.isOpen"
-          v-model="edit.welcomeMessage"
+          :label="$t('GROUP.DESCRIPTION_VERBOSE')"
           icon="fas fa-fw fa-address-card"
-          :label="$t('GROUP.WELCOMEMESSAGE_VERBOSE')"
-          :error="hasError('welcomeMessage')"
-          :error-message="firstError('welcomeMessage')"
+          outlined
+          :tmp-new-border-style="true"
+          class="q-mb-lg"
           @keyup.ctrl.enter="maybeSave"
         />
 
+        <h4 class="text-primary q-mt-xl q-mb-lg">
+          {{ $t('GROUP.HEADINGS.LOCATION') }}
+          <QBtn
+            size="sm"
+            round
+            flat
+            icon="fas fa-info-circle"
+            @click.prevent
+          >
+            <QMenu
+              square
+              dark
+              max-width="280px"
+              content-class="q-pa-md text-center bg-primary"
+            >
+              <p class="text-h5">
+                Extra Info
+              </p>
+              <!-- ToDo -->
+              <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy e</p>
```

Comment:
Obviously needs doing! Maybe @brnsolikyl is up for writing a short text for this and the other one?